package.json

```diff
@@ -21,7 +21,7 @@
     "ember-ajax": "2.5.6",
     "ember-can": "0.8.3",
     "ember-cli": "2.12.0",
-    "ember-cli-app-version": "^2.0.1",
+    "ember-cli-app-version": "^3.0.0",
```

Comment:
Changes to package.json should always be made in individual PRs so we can easily track down bugs if they arise later.  These updates are usually handled automatically by the greenkeeper bot.